content/docs/conditional-rendering.md

```diff
@@ -233,11 +225,13 @@ class Page extends React.Component {
 }
 
 ReactDOM.render(
-  <Page />,
+  <Pagina />,
```

Comment:
```suggestion
  <Page />,
```

Reply:
This is odd @alejandronanez  haha, I had these changes fixed in my local but don't know why they didn't get pushed to remote. Just updated the PR branch. Thanks again. And thanks for the patience reviewing this over and over. My next PR will be much cleaner, promise :)

Reply:
No worries @ElRodrigote, thanks for your hard work here!